docs/labels.md

```diff
@@ -116,3 +118,22 @@ The following command finds all `TaskRuns` that reference a `ClusterTask` named
 ```shell
 kubectl get taskruns --all-namespaces -l tekton.dev/clusterTask=test-clustertask
 ```
+
+## Annotations propagation
+
+Annotation propagate among Tekton entities as follows (similar to Labels):
+
+- For `Pipelines` instantiated using a `PipelineRun`, annotations propagate
+automatically from `Pipelines` to `PipelineRuns` to `TaskRuns`, and then to
+the associated `Pods`. If a label is present in both `Pipeline` and
```

Comment:
s/label/annotation/ 🙃

Reply:
well.. my find-and-replace failed me.. 😹